cicd/build.sh

```diff
@@ -4,30 +4,30 @@
 #IMAGE="quay.io/myorg/myapp" -- docker image URI to push to
 #DOCKERFILE=Dockerfile.custom  -- dockerfile to use (optional)
 #CACHE_FROM_LATEST_IMAGE=true  -- build image from cache from latest image (optional)
-: ${QUAY_EXPIRE_TIME:="3d"}  # sets a time to expire from when the image is built
+: "${QUAY_EXPIRE_TIME:=3d}"  # sets a time to expire from when the image is built
 
 # Env vars set by bootstrap.sh:
 #IMAGE_TAG="abcd123" -- image tag to push to
 #APP_ROOT="/path/to/app/root" -- path to the cloned app repo
 
 # Env vars for local use
-IMAGE_TAG_OPTS="-t ${IMAGE}:${IMAGE_TAG}"
+IMAGE_TAG_OPTS=(-t "${IMAGE}:${IMAGE_TAG}")
 set -e
 
-source ${CICD_ROOT}/_common_container_logic.sh
+# shellcheck source=cicd/_common_container_logic.sh
+source "${CICD_ROOT}/_common_container_logic.sh"
 
 function build {
     if [ ! -f "$APP_ROOT/$DOCKERFILE" ]; then
         echo "ERROR: No $DOCKERFILE found"
         exit 1
     fi
-
     # if this is a PR, set the tag to expire in 3 days
-    if [ ! -z "$ghprbPullId" ] || [ ! -z "$gitlabMergeRequestIid" ]; then
-        echo "LABEL quay.expires-after=${QUAY_EXPIRE_TIME}" >> $APP_ROOT/$DOCKERFILE
+    if [ -n "${ghprbPullId:-}" ] || [ -n "${gitlabMergeRequestIid:-}" ]; then
+        echo "LABEL quay.expires-after=${QUAY_EXPIRE_TIME}" >> "${APP_ROOT}/${DOCKERFILE}"
         # use IMAGE_TAG_LATEST later to detect extra tag to push
-        IMAGE_TAG_LATEST="$(cut -d "-" -f 1,2 <<< $IMAGE_TAG)-latest"
-        IMAGE_TAG_OPTS+=" -t ${IMAGE}:${IMAGE_TAG_LATEST}"
```

Comment:
IMO the old line here using "+=" is more easily understandable compared to the new line 30